developer/hotfix-policy.md

```diff
@@ -0,0 +1,19 @@
+# Hotfix policy
+
+Each scverse core package is maintained by one or multiple core developers who merge pull requests and make releases.
+While all core team members are welcome to contribute to all repos, it is typically the realm of the maintaiers
+to review and merge pull requests. The purpose of this document is to document under which circumstances we
```

Comment:
```suggestion
to review and merge pull requests, as well as to make releases. The purpose of this document is to document under which circumstances we
```

Reply:
In general should this hotfix policy include something about making a release of said package? If we're all on semver maybe only patch releases?